ocf_datapipes/select/select_pv_systems_without_output_wholeday.py

```diff
@@ -40,21 +40,21 @@ def __init__(self, source_datapipe: IterDataPipe) -> None:
     def __iter__(self) -> xr.Dataset():
 
         for xr_dataset in self.source_datapipe:
-            dates_list = xr_dataset.coords["datetime"].values
-            ssid_list = list(xr_dataset)
-            dates_list = [
-                datetime.strptime(str(x)[:10], "%Y-%m-%d").strftime("%Y-%m-%d") for x in dates_list
-            ]
+            sysid = xr_dataset.coords["pv_system_id"].values
+            only_dates = np.asarray(xr_dataset.time_utc.dt.day.values, dtype = int)
+            xr_dataset = xr_dataset.assign_coords(only_dates = ("time_utc", only_dates))
```

Comment:
Good job on changing from the dates list, but I wouldn't assign new time coordinates, ideally just use the time_utc directly for accessing the date.